README.md

```diff
@@ -44,37 +65,38 @@ Plone offers fine-grained permission control over content and actions.
 Plone is easy to set up, extremely flexible,
 and provides you with a system for managing web content that is ideal for project groups, communities, websites, extranets, and intranets.
 
-- *Plone is easy to install.*
+- **Plone is easy to install.**
   Several installation options are available for either your local machine or on servers in the cloud.
 
-- *Plone is easy to use.*
+- **Plone is easy to use.**
   The Plone Team includes usability experts who have made Plone easy and attractive for content managers to add, update, and maintain content.
 
-- *Plone is international.*
+- **Plone is international.**
   The Plone interface has more than 35 translations, and tools exist for managing multilingual content.
 
-- *Plone is standard.*
+- **Plone is standard.**
```

Comment:
"Plone follows standard and is inclusive"

Reply:
follows standards, not standard